src/custom.ts

```diff
@@ -176,14 +168,16 @@ export class CustomExportExtendedPublicKey extends CustomInteraction {
  *
  */
 export class CustomSignMultisigTransaction extends CustomInteraction {
-  /**
-   * @param {object} options - options argument
-   * @param {string} options.network - bitcoin network
-   * @param {array<object>} options.inputs - inputs for the transaction
-   * @param {array<object>} options.outputs - outputs for the transaction
-   * @param {array<string>} options.bip32Paths - BIP32 paths
-   * @param {object} [options.psbt] - PSBT of the transaction to sign, included or generated from the other options
-   */
+  network: BitcoinNetwork;
+
+  inputs: any;
+
+  outputs: any;
+
+  bip32Paths: any;
+
+  psbt: any;
```

Comment:
same here.

Reply:
Readme note added in 8a7d8b3ff59d054336ed85b21940f79453a7f606